### PR TITLE
[Edit] CSS [Edit] CSS Colors: color

### PR DESCRIPTION
**Pull request by ndumidoescode**
### Description

- Made changes to original output
- Made changes to the original example
- Deleted markdown file from previous (now deleted) branch
- Closes issue #3647 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/colors/terms/color/color.md
Comment (dakshdeepHERE) [inline content/css/concepts/colors/terms/color/color.md:37]:
```suggestion
The `h1` tag is set to be orange in color and the `p` tag is set as pink in color: <!--The background color, black/#000000/rgb(0,0,0), is only set for context:
```

Comment (ndumidoescode) [reply content/css/concepts/colors/terms/color/color.md:37]: *Basically what I'm asking is should make the blanked out comment visible to everyone?

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review

File: content/css/concepts/colors/terms/color/color.md
Comment (ishg-153) [inline content/css/concepts/colors/terms/color/color.md:2]:
```suggestion
Title: 'color'
```
This needs to go back to the way it was.

File: content/css/concepts/colors/terms/color/color.md
Comment (ishg-153) [inline content/css/concepts/colors/terms/color/color.md:34]:
```suggestion
## Example
```

File: content/css/concepts/colors/terms/color/color.md
Comment (ishg-153) [inline content/css/concepts/colors/terms/color/color.md:78]: 1. Add a more descriptive `alt-text` (text inside square brackets). Something like this:
`Shows how the color property works with different elements`.
2. The image itself needs to be updated. Change the image so that it only shows the output and not the code.
3. Update the image link (text in round brackets) to this format `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@ndumidoescode I understand that the issue asks you to add media for the example. I have suggested changes concerning that and two other minor changes. Let me know when you are done making these.

**Comment by ndumidoescode** [comment]
> @ndumidoescode I understand that the issue asks you to add media for the example. I have suggested changes concerning that and two other minor changes. Let me know when you are done making these.

@ishg-153 understood, I will work on it. Thank you.

**Comment by ishg-153** [comment]
Hey @ndumidoescode! How are the changes coming along? When should I expect them?

File: content/css/concepts/colors/terms/color/color.md
Comment (ndumidoescode) [reply content/css/concepts/colors/terms/color/color.md]: Hi @ishg-153, I have replaced the image in the media folder, but I am having difficulty linking it to the image link as suggested. Please advise.

Comment (ishg-153) [reply content/css/concepts/colors/terms/color/color.md]: No worries I'll take care of it.

Comment (ishg-153) [reply content/css/concepts/colors/terms/color/color.md]: @ndumidoescode You've used the correct image link, the image doesn't load in your entry until it's merged.

**Review by ishg-153 — APPROVED** [review]
@ndumidoescode I made a few corrections from my end. This is now ready to merge!

**Pull request merged**
